### PR TITLE
Update user map for Nov 2017 #8200

### LINKED ISSUE #8200: Update user map for Nov 2017
Similar to #6765 

New entries to consider:

Australia | Australian National University
-- | --
Australia | Griffith University
Australia | Holmesglen Institute
Australia | University of Melbourne
Australia | University of Western Australia
Belgium | campus de helix
Belgium | Hasselt University
Belgium | Sint Victor Alsemberg
Belgium | Sint-Franciscus College
Belgium | Universiteit Antwerpen
Belgium | Vrije Universiteit Brussel
Belgium | Artevelde University College
Brazil | Universidade do Vale do Rio dos Sinos
Brunei | University Technology Brunei
Canada | École de technologie supérieure
Canada | ETS
Canada | John Abbott College
Canada | Marianopolis College
Canada | New Brunswick Community College
Canada | Okanagan College
Canada | Université du Québec à Montréal
Canada | University of Ottawa
Chile | Universidad de Los Andes
France | École centrale de Nantes
Georgia | Free University
Germany | EBS University
Ghana | Ashesi Universiry College
Hong Kong | The Chinese University of Hong Kong
Hungary | Dózsa György Gimnázium és Táncművészeti Szakgimnázium
Hungary | University of Szeged
India | IIT Bombay
India | JUIT
India | Madanapalle Institute of Technology & Science
India | Thiagarajar College of Engineering
Ireland | NUI Galway
Kenya | Meru University
Malaysia | Monash University Malaysia
Malaysia | Universiti Malaysia Pahang
Malaysia | Universiti Teknologi Malaysia
Malaysia | University of Technology
Mexico | CETYS Universidad
Mexico | Instituto Cumbres Bosques
Mexico | ITESM
Mexico | Universidad de Monterrey
Netherlands | Eindhoven University of Technology
Netherlands | Hogeschool Utrecht
Netherlands | Leiden University
Netherlands | Maastricht University
Netherlands | Saxion
Netherlands | Saxion University of Applied Sciences
Netherlands | University of Applied Sciences of Rotterdam
Netherlands | University of Applied Sciences Utrecht
Netherlands | University of Groningen
New Zealand | Eastern Institute of Technology
New Zealand | MIT Manukau
New Zealand | Mount Albert Grammar School
Philippines | De La Salle University - Dasmariñas
Philippines | University of the Philippines Diliman
Portugal | Instituto Superior Tecnico
Scotland | University of the Highlands & Islands
Singapore | ITE
Singapore | Nanyang Polytechnic
Singapore | Nanyang Technological University
Singapore | Singapore Institute of Technology
Singapore | Singapore Polytechnic
Singapore | Temasek Polytechinc
Sweden | Linnaeus University
UAE | Hult International Business School
UK | Bristol Grammar School
UK | University of Bradford
UK | University of Edinburgh
UK | University of Manchester
UK | University of Salford
UK | University of Suffolk
Ukraine | Sumy State University
Uruguay | Universidad ORT Uruguay
USA | Arizona State University
USA | Brooke Point High School
USA | California State University San Marcos
USA | California State University, San Bernardino
USA | Children's Hospital of Philadelphia
USA | Division Ave High School
USA | Drexel University
USA | Duke University
USA | Duquesne University
USA | Fayetteville State University
USA | Florida Institute of Technology
USA | Florida International University
USA | Francis Tuttle Technology Center
USA | Glendale Community College
USA | Hamilton Mill
USA | Indiana State University
USA | Indiana State University
USA | Indiana University of Pennsylvania
USA | Indiana University of Pennylvania
USA | Iowa State university
USA | Lodi High School
USA | Marywood University
USA | Metropolitain State University of Denver
USA | Miami University
USA | Michigan State University
USA | New York Institute of Technology
USA | North Carolina State University
USA | Northern Arizona University
USA | Ohio University
USA | PSYOP Qualification COurse
USA | Siena College
USA | South Dakota State University
USA | South Plains College
USA | Stony Brook University
USA | SUNY-Albany
USA | Temple University
USA | Texas A&M University
USA | The University of Texas at Austin
USA | Thomas Jefferson University
USA | UNCG
USA | University of California Irvine
USA | University of Dallas
USA | University of Kentucky
USA | University of Minnesota
USA | University of Nevada, Reno
USA | University of St. Augustine
USA | University of Utah
USA | Washington State University
USA | Wellesley College
USA | West Point Middle School
USA | Yavapai College
USA | Alderson Broaddus University

### PR DESCRIPTION
**Pull request by Fatehsandhu**
Fixes #8200

### CODE REVIEW

**Comment by Fatehsandhu** [comment]
@damithc Can you please have a look at this?
Thanks!

**Comment by sukanta-27** [comment]
Before your PR can be reviewed by any core member, some points to consider : 

- Your PR title should be exactly the same as the issue title. (refer to [this](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr))
- You should not open two PRs for the same issue (#8262 and #8263)

**Comment by Fatehsandhu** [comment]
@sukanta-27 It's fixed. 
Thanks!

**Comment by damithc** [comment]
@Fatehsandhu the PR title is not correct. Do read our instructions carefully to find out the exact format required. In addition, you can have a look at all the other open PRs and you will see that your title format is different from others.

**Comment by Fatehsandhu** [comment]
@damithc I fixed the title.

**Comment by Fatehsandhu** [comment]
@damithc Can this be merged now?

**Comment by damithc** [comment]
> @damithc Can this be merged now?

@Fatehsandhu waiting for a developer to review

File: src/main/webapp/data/userMapData.json [hunk 1]
```diff
@@ -579,6 +604,7 @@
       "Centro de Estudios Superiores de Diseño",
       "CETYS Universidad",
       "Escuela Bancaria y Comercial",
+      "Instituto Cumbres Bosques",
       "Instituto Tecnologico de Estudios Superiores",
       "ITESM",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Can incorporate this change too: https://github.com/TEAMMATES/teammates/pull/8204/files#r157783084

File: src/main/webapp/data/userMapData.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "lastUpdated": "End of October 2017",
+  "lastUpdated": "Middle of December 2017",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Use `End of November 2017`

File: src/main/webapp/data/userMapData.json [hunk 3]
```diff
@@ -100,14 +103,15 @@
       "KA Avelgem",
       "KdG University College",
       "KU Leuven",
+      "Sint-Franciscus College",
       "Sint Pieters College",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Might as well fix this for consistency - add a hyphen and move it 2 lines down.

File: src/main/webapp/data/userMapData.json [hunk 4]
```diff
@@ -1629,6 +1679,7 @@
       "Stony Brook University",
       "Suffolk University",
       "Sun Valley High School",
+      "SUNY-Albany",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Same as `State University of New York Albany` earlier in the list.

**Comment by Fatehsandhu** [comment]
@whipermr5 I made all the changes. `Sumy State University` is already in there and `University of Nevada, Reno` & `University of Nevada` are the same.

**Comment by Fatehsandhu** [comment]
@whipermr5 Can this be Merged now?

**Comment by whipermr5** [comment]
@Fatehsandhu Sorry for the delay! The top hit for `University of Nevada` is `University of Nevada, Las Vegas` which is different?

**Comment by Fatehsandhu** [comment]
@whipermr5 No worries. Yes both of them are two separate schools. I will add both though.

**Comment by Fatehsandhu** [comment]
@whipermr5 Anything else you want me to change?
Thanks

**Comment by whipermr5** [comment]
@Fatehsandhu Oh I see; we may need confirmation from @damithc that both schools are really using TEAMMATES. If it's hard to verify, I guess your original solution of sticking to `University of Nevada` might be better 😅

**Comment by damithc** [comment]
> I guess your original solution of sticking to University of Nevada might be better

yes, let's do that.

**Comment by Fatehsandhu** [comment]
@damithc @whipermr5  All done. Anything else I can do? 😁

File: src/main/webapp/data/userMapData.json [hunk 5]
```diff
@@ -26,8 +26,10 @@
       "Edith Cowan University",
       "Federation University",
       "Flinders University of South Australia",
+      "Griffith University",
       "Henley High School",
       "Holy Eucharist School",
+      "Holmesglen Institute",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Just noticed - this should be one line above to follow alphabetical order.

File: src/main/webapp/data/userMapData.json [hunk 6]
```diff
@@ -1410,6 +1456,7 @@
       "Mercer University",
       "Merrimack College",
       "Metropolitan Community College",
+      "Metropolitain State University of Denver",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: Typo in `Metropolitan`.

File: src/main/webapp/data/userMapData.json [hunk 7]
```diff
@@ -1464,6 +1511,7 @@
       "New Prairie United School Corporation",
       "New Tech Odessa High School",
       "New York University",
+      "New York Institute of Technology",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This should be one line above to follow alphabetical order.

File: src/main/webapp/data/userMapData.json [hunk 8]
```diff
@@ -1661,6 +1712,7 @@
       "The University of Tampa",
       "Thiel College",
       "Thomas Ewing Junior High School",
+      "Thomas Jefferson University",
```
Comment (whipermr5) [inline src/main/webapp/data/userMapData.json]: This should be one line below to follow alphabetical order.

Comment (Fatehsandhu) [reply src/main/webapp/data/userMapData.json]: @whipermr5 All done)

**Comment by Fatehsandhu** [comment]
@whipermr5 Thanks, It was nice to work with you!

**Pull request merged**